### PR TITLE
docs/dev:libvirt Add section to change mem/cpu for master/worker for libvirt

### PR DESCRIPTION
**Pull request by praveenkumar**
This PR is to test libvirt CI break.

- https://github.com/openshift/installer/pull/2399

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
/close

has been DONOTMERGE for some time now.

File: docs/dev/libvirt/customization.md [hunk 1]
```diff
@@ -20,3 +20,45 @@ platform:
...
+
+## Change cpu/memory
+
+- Create manifests
+
+```bash
+$ openshift-install --log-level=debug --dir=. create manifest
+```
+
+- Edit openshift/99_openshift-cluster-api_master-machines-0.yaml to read (only relevant part shown)
```
Comment (gbraad) [inline docs/dev/libvirt/customization.md:32]: s/to read/as follows

File: docs/dev/libvirt/customization.md [hunk 2]
```diff
@@ -20,3 +20,45 @@ platform:
...
+spec:
+  template:
+    spec:
+      providerSpec:
+        value:
+          apiVersion: libvirtproviderconfig.openshift.io/v1beta1
+          domainMemory: 16384
+          domainVcpu: 6
+```
+
```
Comment (cfergeau) [inline docs/dev/libvirt/customization.md:59]: Why show both files? should I always set worker memory to twice the memory of the master?
You could explicitly say that first one is to change the memory/vcpus for the master nodes, and that the 2nd one is for the workers, and give some guidelines for the minimal values to use. Mention that memory is in kB

File: docs/dev/libvirt/customization.md (hunk 1 above)
Comment (cfergeau) [reply docs/dev/libvirt/customization.md:32]: Still seems to apply

File: docs/dev/libvirt/customization.md [hunk 3]
```diff
@@ -20,3 +20,45 @@ platform:
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ```
+
+## Change cpu/memory
+
+- Create manifests
+
+```bash
+$ openshift-install --log-level=debug --dir=. create manifest
```
Comment (vrutkovs) [inline docs/dev/libvirt/customization.md:29]:
```suggestion
$ openshift-install --log-level=debug --dir=. create manifests
```

**Pull request closed without merging**